EIC-EA/EIC-community-D8
# Feature/eicnet 904 group wiki styling up to date

Tests:

* Create a group with a wiki
* **You should see the styled "no wiki pages" message**
* Create a wiki page
* **You should see the styled wiki page**



* Create a book page
* **You should see the styled book page**

Unfinished:

* The content of the wiki and book page is not displayed as in storybook, but the html is a one on one match.
